package.json

```
@@ -99,7 +99,8 @@
     ],
     "activationEvents": [
       "onLanguage:chapel",
-      "onLanguage:chapel-ast"
+      "onLanguage:chapel-ast",
+      "workspaceContains:*.chpl"
```

Based on my understanding, the current glob pattern only matches chapel files at the top level of the workspace. To match files in sub-folders:

```suggestion
      "workspaceContains:**/*.chpl"
```